### DISCUSSION TITLE
Expand row with click-row event

### DISCUSSION THREAD

**Discussion by marmeni** [HC200ok/vue3-easy-data-table#152] [Q&A] [answered]
Is there any way to expand/close a row using the click-row event? If not, are there any plans to implement this feature in the future?

**Comment by HC200ok** [comment]
We already have the `Expandable row` feature, Could you tell me why you need click-row event to expand/close a row?

**Reply by marmeni** [reply]
We were looking to be able to expand the row by simply clicking anywhere on the row rather than the just the column with the little arrow.

**Comment by HC200ok** [comment, marked as answer]
@marmeni please update to the latest version `1.5.14`, by passing the prop `click-row-to-expand` you can click a row to expand.
```
<DataTable
   :headers="headers"
   :items="items"
   click-row-to-expand
</DataTable>
```

**Reply by marmeni** [reply]
Works like a charm. Thanks so much!

**Comment by matrixmandeep** [comment]
Hey is there any way to expand the row on particular cell click?

**Reply by yashwp** [reply]
@HC200ok This is very necessary!! We have use-case like this and there is no way to achieve that at the moment.

<img width="623" alt="image" src="https://github.com/HC200ok/vue3-easy-data-table/assets/13454790/d0bd5810-b54c-42cb-a48f-1ac09bfdaf56">
